### DISCUSSION TITLE
How to deploy a MAUI app on Linux?

### DISCUSSION THREAD

**Discussion by RadoslawJasinski** [dotnet/maui#10903] [Q&A]
Hi,
Does MAUI allow you to write applications for Linux?
I tied deploy app using command: 
`dotnet publish -c release -r linux-x64`

but I have got a error:
```
error NETSDK1082: There was no runtime pack for Microsoft.Maui.Core available for the specified RuntimeIdentifier 'linux-x64'
error NETSDK1082: There was no runtime pack for Microsoft.Maui.Controls available for the specified RuntimeIdentifier 'linux-x64'. 
error NETSDK1082: There was no runtime pack for Microsoft.Maui.Essentials availablefor the specified RuntimeIdentifier 'linux-x64'. 
``` 
Does MAUI support Linux?

**Comment by manishbisen** [comment]
I think yet Maui doesn't support linux

**Comment by xtuzy** [comment]
Why you not read document https://learn.microsoft.com/en-us/dotnet/maui/supported-platforms.

Current only one of community to develop it https://github.com/jsuarezruiz/maui-linux/pull/37

**Comment by MartyIX** [comment]
https://github.com/dotnet/maui/discussions/339 is related to this question.
